wagtail/wagtailimages/models.py

```diff
@@ -338,7 +344,10 @@ def get_image_model():
 
     image_model = apps.get_model(app_label, model_name)
     if image_model is None:
-        raise ImproperlyConfigured("WAGTAILIMAGES_IMAGE_MODEL refers to model '%s' that has not been installed" % settings.WAGTAILIMAGES_IMAGE_MODEL)
+        raise ImproperlyConfigured(
+            "WAGTAILIMAGES_IMAGE_MODEL refers to model '%s' that has not been installed"
+            % settings.WAGTAILIMAGES_IMAGE_MODEL
```

Comment:
The % operator should be on the previous line according to https://www.python.org/dev/peps/pep-0008/#maximum-line-length "The preferred place to break around a binary operator is after the operator, not before it"